src/components/DpStickyElement/DpStickyElement.stories.mdx

```diff
@@ -0,0 +1,32 @@
+import { Meta } from '@storybook/addon-docs'
+import DpStickyElement from './DpStickyElement'
+
+<Meta
+    title="Components/StickyElement"
+    component={DpStickyElement}
+/>
+
+# StickyElement
+
+The StickyElement component is used to create elements that stick to a specific position (top or bottom) while scrolling.
+
+## Default usage
+
+```html
+<dp-sticky-element
+  direction="bottom"
+  ref="footer"
+  class="c-mastertoeb__footer">
+  <div class="u-mv-0_5 text-right">
+    <sliding-pagination
+      :current="currentPage"
+      :total="totalPages"
+      @page-change="handlePageChange" />
+    <dp-select-page-item-count
+      @changed-count="setPageItemCount"
+      :page-count-options="itemsPerPageOptions"
+      :current-item-count="itemsPerPage"
+      :translations="{ pagerElementsPerPage: Translator.trans('pager.per.page') }" />
+  </div>
+</dp-sticky-element>
```

Comment:
For the code examples in general, i'd suggest to only use the minimal code that is needed to get this component working. In this case, the `ref` and `class` attributes are not needed for the example, as they may distract readers.

If a code example needs additional components to illustrate usage, i suggest to also keep that example markup to a bare minimum: only use native Html elements, or components from demosplan-ui. Also i feel that those "placeholder components" should be reduced to their bare minimum. In the example here, we pass a div into the slot, which contains two rather complex components. In my opinion, it would be enough to have two button elements (maybe "Save" and "Cancel"), to illustrate the case of a sticky save bar. It would also be ok to omit the example content at all, by only placing a Html comment inside like `<!-- Whatever you place here is gonna stick to the bottom -->`.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/491/commits/947a8cc7bf1ff2b7c4b02134759a4d2f496973a9
Thanks!